src/main/java/com/solvd/webtest/components/ResultMobile.java

```diff
@@ -0,0 +1,84 @@
+package com.solvd.webtest.components;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.time.Duration;
+
+public class ResultMobile extends ResultBase {
+    @FindBy(className = "s-item__title")
+    private ExtendedWebElement title;
+
+    @FindBy(className = "s-item__price")
+    private ExtendedWebElement price;
+
+    @FindBy(className = "s-item__location")
+    private ExtendedWebElement location;
+
+    @FindBy(className = "s-item__shipping")
+    private ExtendedWebElement shipping;
+
+    @FindBy(css = ".s-item__sep > span")
+    private ExtendedWebElement sponsored;
+
+    public ResultMobile(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+    public String getItemTitle() {
+        if (title.isElementPresent(Duration.ofMillis(100)))
```

Comment:
What if element might take longer to load?

Reply:
Is there a way to do it better?